cimpl/ecqm_dataelement.txt

```diff
@@ -4294,7 +4294,7 @@ Subject value is type HospitalObservationCareInitial
 
 EntryElement: HospitalSettingsAdmissionSource
 Based on: AdmissionSource
-Description: "(Clinical Focus: This set of values identifies hospital environments where a patient may receive inpatient or outpatient care.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care.  Using the Quality Data Model, this particular element will map to the Transfer category.),(Inclusion Criteria: Use the concept of hospital environment (22232009) and all its children.  Codes used are to be SNOMED CT codes only.),(Exclusion Criteria: None.)"
+Description: "(Clinical Focus: This set of values identifies hospital environments where a patient may receive inpatient or outpatient care.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care. Using the Quality Data Model, this value set is used with the \"Admission Source\" attribute of the \"Encounter\" category.),(Inclusion Criteria:  Include codes that represent the concept of hospital environment. Codes included are SNOMED CT 22232009 and all its children. ),(Exclusion Criteria: None.)"
```

Comment:
The SHR toolchain doesn't like the escaped double quotes, so we'll probably have to change them to single quotes.
```suggestion
Description: "(Clinical Focus: This set of values identifies hospital environments where a patient may receive inpatient or outpatient care.),(Data Element Scope: The intent of this data element is to identify hospital environments where a patient may receive inpatient or outpatient care. Using the Quality Data Model, this value set is used with the 'Admission Source' attribute of the 'Encounter' category.),(Inclusion Criteria:  Include codes that represent the concept of hospital environment. Codes included are SNOMED CT 22232009 and all its children. ),(Exclusion Criteria: None.)"
```